src/_utils/branding.ts

```diff
@@ -144,11 +144,13 @@ export const responsiveBreakpoints = {
 
 export const pxToInteger = (size: string): number => parseInt(size.replace('px', ''), 10)
 
-export default {
+export const branding = {
```

Comment:
This export has been kept because a `branding` object is exported from the package entry point.